src/pages/workspace/WorkspaceInitialPage.js

```diff
@@ -109,7 +110,7 @@ class WorkspaceInitialPage extends React.Component {
             {
                 translationKey: 'workspace.common.bankAccount',
                 icon: Expensicons.Bank,
-                action: () => Navigation.navigate(ROUTES.getWorkspaceBankAccountRoute(policy.id)),
+                action: () => WorkSpaceUtils.getShouldShowPage(this.props, ROUTES.getWorkspaceBankAccountRoute(policy.id), ROUTES.getBankAccountRoute()),
```

Comment:
All of these statements have a lot of duplicate code. Shouldn't we move that inside the function? 

Also, do not pass `this.props`. Pass only necessary data to the function which is really needed.

Reply:
Done